lib/twilio-ruby/jwt/access_token.rb

```diff
@@ -144,6 +144,31 @@ def payload
         end
       end
 
+      class ConversationsGrant
+        attr_accessor :configuration_profile_sid
+
+        def key
+          'rtc'
+        end
+
+        def payload
+          payload = {}
+
+          if configuration_profile_sid
+            payload[:configuration_profile_sid] = configuration_profile_sid
+          end
+
+          payload
+        end
+
+       class << self
+         extend Gem::Deprecate
```

Comment:
It is part of standard library, but I am not a ruby expert. We use language constructs in other helper libraries as well.